plugins/Flanger/FlangerControlsDialog.cpp

```diff
@@ -38,24 +38,24 @@ FlangerControlsDialog::FlangerControlsDialog( FlangerControls *controls ) :
 	QPalette pal;
 	pal.setBrush( backgroundRole(), PLUGIN_NAME::getIconPixmap( "artwork" ) );
 	setPalette( pal );
-	setFixedSize( 200, 75 );
+	setFixedSize( 195, 75 );
 
 	Knob* delayKnob = new Knob( knobBright_26, this );
-	delayKnob->move( 20,10 );
+	delayKnob->move( 10,10 );
 	delayKnob->setVolumeKnob( false );
 	delayKnob->setModel( &controls->m_delayTimeModel );
 	delayKnob->setLabel( tr( "Delay" ) );
 	delayKnob->setHintText( tr( "Delay Time:" ) + " ", "s" );
 
 	TempoSyncKnob * lfoFreqKnob = new TempoSyncKnob( knobBright_26, this );
-	lfoFreqKnob->move( 53,10 );
+	lfoFreqKnob->move( 45,10 );
 	lfoFreqKnob->setVolumeKnob( false );
 	lfoFreqKnob->setModel( &controls->m_lfoFrequencyModel );
 	lfoFreqKnob->setLabel( tr( "Lfo Hz" ) );
 	lfoFreqKnob->setHintText( tr ( "Lfo:" ) , "Hz" );
 
 	Knob * lfoAmtKnob = new Knob( knobBright_26, this );
-	lfoAmtKnob->move( 86,10 );
+	lfoAmtKnob->move( 84,10 );
 	lfoAmtKnob->setVolumeKnob( false );
 	lfoAmtKnob->setModel( &controls->m_lfoAmountModel );
 	lfoAmtKnob->setLabel( tr( "Amt" ) );
```

Comment:
Thank you @zonkmachine  @tresf @jasp00 for the input. From reading over the comments here I'm not sure whether we have come to a final decision about keeping it as AMT or changing it to AMNT?